packages/cozy-client/src/associations/HasMany.js

```diff
@@ -256,6 +272,50 @@ export const setHasManyItem = (HasMany.setHasManyItem = (
     relName,
     relationship => merge({}, relationship, { data: updatedRelData })
   )
+
+  return updatedDocument
+})
+
+/**
+ * Sets multiple relationships with their respective name and ID
+ *
+ * @param {object} doc - Document to be updated
+ * @param {Array.<Relationship>} relationships - Array of relationships to be sets
+ */
+export const setHasManyItems = (HasMany.setHasManyItems = (
```

Comment:
I'm wondering about the utility and naming of this function. 

* It seems like the symetric setter of getHasManyItems but in fact it sets *multiple relationships* whereas getHasManyItems returns multiple *items* of a single relationship. This makes it asymetrical to getHasManyItems and this can be a source of confusion.
To me, the symetrical of getHasManyItems would be setHasManyItems(doc, relName, items).

* I think it is unnecessary to create a copy of updatedDocument to be coherent with the single setHasManyItem function.

* It creates a new API in the form of the { relName, relItemId, relItemAttrs } type. 

* I am not sure about the utility of abstracting a forEach here.

At this point, I am not sure that this function should be included.

Reply:
Yes indeed, I had created it for a personal need, but I'm not sure it's pertinant to integrate it into Cozy-Client
It has no real added value